### PR TITLE
[MIRROR] [MIRROR] Adds Gibberish

### PR DESCRIPTION
**Pull request by CitadelStationBot**
Original PR: https://github.com/VOREStation/VOREStation/pull/4688
--------------------
Original PR: https://github.com/PolarisSS13/Polaris/pull/5843
--------------------
Screwing up on a language key now defaults to a specific language, called Babel, pathed as /datum/language/gibberish. Means that people specifically ***won't*** understand what you say if you do screw up. Neither will translators or recorders. I like to imagine it as the sounds that come out of one's mouth when they lose their train of though whilst really tired and just keep rambling on.

TL;DR:
No more giving away your plans with ",5Yeah, let's kill the meatbags!".
Now it turns into "Bleargh Oum Blem!" or something equally silly.

### CODE REVIEW

**Comment by nik707** [comment]
From before hardsync began
